tests/test_struct.py

```diff
@@ -161,3 +163,146 @@ def test_raw_dump_size(self):
             self.assertEqual(obj.get_size(), unpacked.get_size())
         except FileNotFoundError:
             raise self.skipTest('No raw dump file found.')
+
+
+class TestStructDump(unittest.TestCase):
+    """Run message pack, unpack and get_size tests using bytes struct dump.
+
+    Test the lib with raw dumps. We assume the raw files are valid according
+    to the OF specs to check whether our pack, unpack and get_size
+    implementations are correct.
+
+    Also, check the minimum size of the struct by instantiating an object with
+    no parameters.
+
+    To run the tests, just extends this class and set the 'dump' and 'obj'
+    attributes. You can also optionally set the 'min_size' attribute.
+
+    Example:
+        .. code-block:: python3
+
+            class TestMatch(TestStructDump):
+                dump = b'' # needs to be filled
```

Comment:
someone may want to write the dump attribute in the setup class.
no reason to forbid it.
I'll add the extra space for the comment inside the comment that is the docstring.....